Subject: [PATCH] make microcode text less confusing
Date:	Mon, 27 Dec 2004 06:20:19 +0000
X-BK-ChangeSetKey: alan@lxorguk.ukuu.org.uk[torvalds]|ChangeSet|20041227062019|28804
Patch-mainline: yes
From: olh@suse.de


ChangeSet 1.2190, 2004/12/26 22:20:19-08:00, alan@lxorguk.ukuu.org.uk

	[PATCH] make microcode text less confusing
	
	Red Hat got some confused customers due to this message. The confused
	user case is when they update the BIOS and all of a sudden we have "no
	suitable data" yet we did before. We (Arjan van de Ven) thus changed it
	to "No new microcode" which is much much clearer.
	
	Signed-off-by: Alan Cox <alan@redhat.com>
	Signed-off-by: Linus Torvalds <torvalds@osdl.org>



 microcode.c |    2 +-
 1 files changed, 1 insertion(+), 1 deletion(-)


diff -Nru a/arch/i386/kernel/microcode.c b/arch/i386/kernel/microcode.c
--- a/arch/i386/kernel/microcode.c	2004-12-26 23:26:54 -08:00
+++ b/arch/i386/kernel/microcode.c	2004-12-26 23:26:54 -08:00
@@ -364,7 +364,7 @@
 	struct ucode_cpu_info *uci = ucode_cpu_info + cpu_num;
 
 	if (uci->mc == NULL) {
-		printk(KERN_INFO "microcode: No suitable data for CPU%d\n", cpu_num);
+		printk(KERN_INFO "microcode: No new microcode data for CPU%d\n", cpu_num);
 		return;
 	}
 
-
To unsubscribe from this list: send the line "unsubscribe bk-commits-head" in
the body of a message to majordomo@vger.kernel.org
More majordomo info at  http://vger.kernel.org/majordomo-info.html

